### ISSUE TITLE
[minor] tests are glibc specific

### ISSUE THREAD

**Issue by loreb** [cuadue/2048_game#9]
In case anyone is running this anywhere else (*BSD, mac, distros using musl/...) the tests
will fail because srandom() can be (and is) implemented as the authors of the C library wish.

Simplest solutions:
- ignore
- just warn :)
- use rand48 (worse than random, so maybe only for "-s", but it's the only standard prng which specifie BOTH the generator AND the seeding procedure)

**Comment by cuadue** [comment]
Good call. Might as well just drop in a simple linear feedback shift register and eschew `srandom` and `random` altogether.

**Comment by loreb** [comment]
If you're looking to include a generator, http://xoroshiro.di.unimi.it/ has a nice collection,
with explanations (or links to them), comparisons & implementations (mostly GPLv3)

ISAAC (http://burtleburtle.net/bob/rand/isaac.html) is public domain

Knuth (https://cs.stanford.edu/~knuth/news02.html#rng) is public domain as long as you make no changes
